CMakeLists.txt

```diff
@@ -280,6 +280,13 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND
   add_gcc_cpp_compile_options("-Wno-noexcept-type")
 endif()
 
+find_package(PythonInterp)
+if (!PYTHONINTERP_FOUND)
+  message(FATAL_ERROR "Can not find python.")
+endif()
+
+message("Using python: ${PYTHON_EXECUTABLE}")
```

Comment:
Add to the previous block?
if (found) print(found) else error()
?

Reply:
done